### ISSUE TITLE
Problem with typing promo code on checkout and shopping cart page.

### ISSUE THREAD

**Issue by 321-Soleil** [woocommerce/woocommerce-gutenberg-products-block#3761]
Hello i found a little problem on the checkout and on the shopping cart page, when i activate the promo code for my site with woocommerce and  go to the cart page and click on the promo code button, it does something litlle weird because it does not show me what i am typing. furthermore, when i activate it it say "remise" which is good but after that it says"and NBSP" which is not so good and finally, when i type with my phone, if i type "321Soleil", shopping cart shows :" lieloS123".

[##]([url](url
![Capture d’écran (149)](https://user-images.githubusercontent.com/76965021/106164395-335c4c00-618a-11eb-914c-1d7151fca634.png)
)) Describe the bug

### WordPress (please complete the following information):
* Core version: [e.g. 5.0.3]
* Gutenberg version (if installed): [e.g. 4.8.0]
* WooCommerce version: [e.g. 3.5.3]
* WooCommerce Blocks version: [e.g. 1.3.1]
* Site language:
* Any other plugins installed:

### Desktop (please complete the following information):
* OS: [e.g. macOS, Windows]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

### Smartphone (please complete the following information):
* Device: [e.g. iPhone6]
* OS: [e.g. iOS8.1]
* Browser [e.g. stock browser, safari]
* Version [e.g. 22]

**Comment by budzanowski** [comment]
Hello,
Before we can find out what is happening we would like to ask you to provide additional information:
What version of WordPress and WooCommerce are you testing with. Do you have the WooCommerce Blocks or Gutenberg plugin installed?
What type of browser are you using and what type of Smartphone ( Device type, OS, Browser ) are you using?
What is the site language settings?

I have visited https://321soleil.fr/cart/ and I have not seen the promo code section. Is it currently enabled?

**Comment by 321-Soleil** [comment]
Hello, Yes i have Disabled the promo codes do you want me to enable them ?
I use A Samsung galaxy 9 Smartphone, my language is in French, yes it is because i use the Woocommerce block plug in. I think i have the new version of wordpress
________________________________
De : Bartosz Budzanowski <notifications@github.com>
Envoyé : vendredi 29 janvier 2021 10:26
À : woocommerce/woocommerce-gutenberg-products-block <woocommerce-gutenberg-products-block@noreply.github.com>
Cc : 321-Soleil <t1delage@hotmail.fr>; Author <author@noreply.github.com>
Objet : Re: [woocommerce/woocommerce-gutenberg-products-block] Hello i found a little problem on the checkout and on the shopping cart page, when i activate the promo code for my site with woocommerce and go to the cart page and click on the promo code button...


Hello,
Before we can find out what is happening we would like to ask you to provide additional information:
What version of WordPress and WooCommerce are you testing with. Do you have the WooCommerce Blocks or Gutenberg plugin installed?
What type of browser are you using and what type of Smartphone ( Device type, OS, Browser ) are you using?
What is the site language settings?

I have visited https://321soleil.fr/cart/ and I have not seen the promo code section. Is it currently enabled?

**Comment by opr** [comment]
Hi @321-Soleil - it would be helpful to see your system status report - You will find information on how to do this here: https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/#section-1

If you can paste the status report into this thread that would be great.

If possible, please enable the promo codes again so we may take a look to see if this is an issue being caused by your theme's styling or whether it is a bug with our extension.

Thanks!

**Comment by ralucaStan** [comment]
Duplicate of  #3804, moving discussion there.
